sycl/source/exception.cpp

```diff
@@ -26,5 +26,15 @@ context exception::get_context() const {
 
 cl_int exception::get_cl_code() const { return MCLErr; }
 
+const detail::SYCLCategory SYCLCategoryObj;
```

Comment:
Can you please double check, that this thing does not create problems with global constructors? clang suggests that it's not a POD: https://godbolt.org/z/KeoY51E33

Reply:
Nevermind, I don't see constructor to add new assembly to the result: https://godbolt.org/z/MGPz4rnTM